docs/reference/commandline/build.md

```diff
@@ -457,3 +459,123 @@ space.
 storing two copies of the image, one for the build cache with all the cache
 layers in tact, and one for the squashed version.
 
+#### Prerequisites
+
+The example on this page is using experimental mode in Docker 1.13.
+
+Experimental mode can be enabled by using the `--experimental` flag when starting the Docker daemon or setting `experimental: true` in the `daemon.json` configuration file.
+
+By default, experimental mode is disabled. To see the current configuration, use the `docker version` command.
+
+```
```

Comment:
After the three back-ticks, put the word `none`. Imagine that these are back-ticks: `'''none`

This suppresses syntax highlighting, which won't work well here.